### ISSUE TITLE
[Bug] Graphical glitch in bass enhancer

### ISSUE THREAD

**Issue by tim-gromeyer** [wwmm/easyeffects#2780]
### EasyEffects Version

7.0.7

### What package are you using?

Other (specify below)

### Distribution

Kubuntu 23.10

### Describe the bug

There is a graphical bug in the Bass Enhancer, see the screenshot below:

![image](https://github.com/wwmm/easyeffects/assets/58736434/3f3d26ac-5a2e-49f0-a2cb-e10dffe29db1)

The bar behind "Obertöne" (in english Overtones) doesn't fit the window size, I don't know how to describe it, just have a look.

Note: This also happens on my main computer running openSUSE Tumbleweed, the very latest version.

On this system (a VM at work) I use the Ubuntu package.

### Expected Behavior

I don't know how to describe it

### Debug Log

None

**Comment by wwmm** [comment]
This is a known gtk4 bug #1441. They are aware of it but for some reason years are passing and the bug is still there... I think that at some point of of their developers started working on a fix but for some reason it never got to the stable releases.

**Comment by tim-gromeyer** [comment]
@wwmm Does it just overflow when the max value is excited? If so, it's only a matter of 1 std::min

**Comment by wwmm** [comment]
>Does it just overflow when the max value is excited? If so, it's only a matter of 1 std::min

Yes. We could workaround it. It would be annoying because not all of our GtkLevelBar have `[0,1]` as boundaries. So we have to carefully look at each case and customize the workaround. Just to fix something that never happened in gtk3...

The thing is that it makes no sense that gtk4 allows a widget to draw things over another's surface. It is a fundamental flaw they have to fix. It is not because the GtkLevelBar received a value outside of its min/max range that it should draw things over other widgets. That is what the pull requests someone submitted there is avoiding. They just have to merge the thing.
